DocGen4/Output/Module.lean

```diff
@@ -195,14 +199,23 @@ def internalNav (members : Array Name) (moduleName : Name) : HtmlM Html := do
 /--
 The main entry point to rendering the HTML for an entire module.
 -/
-def moduleToHtml (module : Process.Module) : HtmlM Html := withTheReader SiteBaseContext (setCurrentName module.name) do
+def moduleToHtml (module : Process.Module) (backrefs : Array BackrefItem) :
+    HtmlM (Html × Array BackrefItem) := withTheReader SiteBaseContext (setCurrentName module.name) do
   let relevantMembers := module.members.filter Process.ModuleMember.shouldRender
-  let memberDocs ← relevantMembers.mapM (moduleMemberToHtml module.name)
+  let mut memberDocs : Array Html := #[]
+  let mut newBackrefs := backrefs
+  let mut idx : Nat := 0
+  while h : LT.lt idx relevantMembers.size do
```

Comment:
Please use proper `<` syntax instead of this, dito at the other places where you have `LT.lt`

Reply:
The actual problem is that the `open scoped Jsx` breaks `<` parsing in expressions 😅 so I have to use `LT.lt`.